docs/view-flattening.md

```diff
@@ -3,15 +3,17 @@ id: view-flattening
 title: View Flattening
 ---
 
-> This document refers to the architecture of the the new renderer, [Fabric](fabric-renderer), that is in active roll-out.
+> This document refers to the architecture of the new renderer, [Fabric](fabric-renderer), that is in active roll-out.
 
 #### View Flattening is an optimization by the React Native renderer to avoid deep layout trees.
 
-The React API is designed to be declarative and reusable through composition; this is great for product developers because they can focus on what they want to render instead of how to render on the screen. However, in practice, these qualities of the API lead to the creation of deep React Element Trees, where a large majority of their Nodes only affect the layout of a View and don’t render anything on the screen, we call these types of nodes “Layout-Only” Nodes.
+The React API is designed to be declarative and reusable through composition. This provides a great model for intuitive development. However, in implementation, these qualities of the API lead to the creation of deep [React Element Trees](glossary#react-element-tree-and-react-element), where a large majority of React Element Nodes only affect the layout of a View and don’t render anything on the screen. We call these types of nodes **“Layout-Only”** Nodes.
```

Comment:
why "in implementation"? I'm not familiar with this. Does it mean the same as "in practice"? 
I tried to google the term but didn't find anything relevant.

Reply:
I like how you split this into multiple sentences.